### DISCUSSION TITLE
Winget package ?

### DISCUSSION THREAD

**Discussion by SeleDreams** [dlang/dub#2712] [Ideas]
Hi,
I was thinking, would it be possible to make a winget package of dub ? I think it would make it quicker to get started with the dub package manager on windows to simply have to type "winget install dub" and easily update it over time the same way.

**Comment by SeleDreams** [comment]
Actually, being new to D, I assumed dub was standalone. (I usually worked
in C and C++) I noticed DMD does have a winget package, does it bundle dub
in this case ?

**Reply by rikkimax** [reply]
Winget build of dmd is not up to date.

https://winget.run/pkg/Dlang/DMD

I would suggest to use the installers or the 7z archive on D's website instead which does bundle dub.

**Comment by rikkimax** [comment]
What is the scenario where you have a D compiler, but no dub binary?
